src/include/sof/alh.h

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2019 Intel Corporation. All rights reserved.
+//
+// Author: Slawomir Blauciak <slawomir.blauciak@linux.intel.com>
+
+#ifndef __INCLUDE_ALH__
+#define __INCLUDE_ALH__
+
+#define SOF_ALH_H
```

Comment:
Could we start a naming pattern here and maybe name it exactly like guard, but without underscores?
`#define SOF_DRIVERS_ALH_H`